swimster-ui/src/components/App/App.jsx

```diff
@@ -32,17 +33,14 @@ const App = () => {
 
     initApp()
     console.log("App is ready to go!")
-  }, [setInitialized, isUserAuthenticated])
+  }, [setInitialized, isAuthenticated])
 
   return (
     <BrowserRouter>
       <main className={style.app}>
         <Navbar />
         <Routes>
-          <Route path='/' element={
-            <ProtectedRoute element={<Home />} />
-          }
-          />
+          <Route path='/' element={<Home />} />
```

Comment:
why did you take the protected route out?

Reply:
It was giving me a bug when rendering listings on first load, referenced in this [commit](https://github.com/Facebook-University-Projects/Swimster/pull/9/commits/e7c680f9ac69603a941fa5158dc58a8a20c80439)

Reply:
I was able to solve the issue by redirecting non-logged in users to `/login`, like so
https://github.com/Facebook-University-Projects/Swimster/blob/e67c4a57c5f1de64ad7981c04cf9021410b2cc77/swimster-ui/src/components/Navbar/Navbar.jsx#L13-L17